roles/ceph-rgw/tasks/rgw_create_pools.yml

```diff
@@ -0,0 +1,71 @@
+---
+  - name: remove ec profile
+    command: "{{ container_exec_cmd }} ceph --connect-timeout 5 --cluster {{ cluster }} osd erasure-code-profile rm {{ item.ec_profile }}"
```

Comment:
because we're using a dict lookup the values need to use `item.value.<value>`
So it should be `item.value.ec_profile`